lib/active_record/tasks/sqlserver_database_tasks.rb

```diff
@@ -70,11 +70,14 @@ def structure_dump(filename, extra_flags)
         dump.gsub!(/^GO\n/, "")                               # Strip db GO statements
         dump.gsub!(/nvarchar\(8000\)/, "nvarchar(4000)")      # Fix nvarchar(8000) column defs
         dump.gsub!(/nvarchar\(-1\)/, "nvarchar(max)")         # Fix nvarchar(-1) column defs
+        dump.gsub!(/varbinary\(-1\)/, "varbinary(max)")       # Fix varbinary(-1) column defs
```

Comment:
👀 came here for this

Reply:
Does this fix a bug you were having with a specific version of Rails (6.0/6.1/etc)? 

Would you be able to update `test/cases/rake_test_sqlserver.rb` to demonstrate the fix?

Reply:
yes, wil do.

Reply:
Not a Rails version issue. Just some database we're querying on.

Reply:
Can you maybe add that test case?